modules/apps/27-interchain-accounts/controller/keeper/msg_server_test.go

```diff
@@ -85,3 +92,108 @@ func (suite *KeeperTestSuite) TestRegisterAccount() {
 		}
 	}
 }
+
+func (suite *KeeperTestSuite) TestSubmitTx() {
+	var (
+		path         *ibctesting.Path
+		owner        string
+		connectionID string
```

Comment:
do we need to expose this var to mutate, or can we just use `path.EndpointA.ConnectionID` and mutate the path in `malleate()`?

Reply:
I think it's fine to leave as is :handshake:

Reply:
You're right actually, I updated it. Thanks